src/Sulu/Bundle/WebsiteBundle/Resources/views/Sitemap/sitemap.xml.twig

```diff
@@ -1,42 +1,12 @@
 {% spaceless %}
 <?xml version="1.0" encoding="UTF-8"?>
-<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+<urlset xmlns:xhtml="http://www.w3.org/1999/xhtml" {% block namespaces %}xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"{% endblock %}>
     {% set registrableDomain = sulu_util_domain_info(domain).registrableDomain %}
 
     {% for entry in entries %}
         {% set location = sulu_content_path(entry.loc, webspaceKey, entry.locale, null, scheme) %}
         {% if sulu_util_domain_info(location).registrableDomain == registrableDomain %}
-            <url>
```

Comment:
would just add a blocks here instead of using includes. as its better when you overwrite the sitemap.xml you have to overwrite only one template